### PR TITLE
add support for blacklight range limit

### LINKED ISSUE #147: Range limit plugin is broken
Update to BL 6 borked the range limit plugin. @tampakis has experience with getting this working.

### PR DESCRIPTION
**Pull request by tampakis**
Current released version of range limit gem doesn't support rails 5.1. Pinned to a branch of the project that is compatible with 5.1. Closes #147.

### CODE REVIEW

**Comment by tpendragon** [comment]
@tampakis Nice!

**Comment by tampakis** [comment]
i should really run rubocop before submitting PRs though...

File: .byebug_history [hunk 1]
```diff
@@ -0,0 +1 @@
+c
```
Comment (escowles) [inline .byebug_history]: Can you add `.byebug_history` to `.gitignore`?

File: Gemfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ gem 'bitters', '~> 1.2.0'
 gem 'bixby'
 gem 'blacklight', '~> 6.10'
 gem 'blacklight-gallery', '~> 0.4'
-gem 'blacklight_range_limit', '~> 6.1'
+gem 'blacklight_range_limit', git: 'https://github.com/projectblacklight/blacklight_range_limit', branch: 'rails-5.1'
```
Comment (hackmastera) [inline Gemfile]: Can you please add a comment explaining why this branch is required and what conditions should be met to go back to a release?

**Pull request merged**